core/embed/lib/unit_variant.c

```diff
@@ -34,3 +65,20 @@ uint8_t unit_variant_get_color(void) { return unit_variant_color; }
 bool unit_variant_get_btconly(void) { return unit_variant_btconly; }
 
 bool unit_variant_present(void) { return unit_variant_ok; }
+
+bool unit_variant_is_sd_hotswap_enabled(void) {
```

Comment:
Just a comment: unix OTP (mocked for emu) is currently programmed to contain all `1`, resulting in `unit_variant_get_build_year()` returning `-1` (no dash found)  for emulator, i.e. emulator won't be "hot-swappable". 

I will probably mock different OTP for emu in #3441 to simulate both cases.

Reply:
Dev boards have OTP also empty, but seem to be hot swappable in HW.
Though not sure if QA still uses some of the dev boards still, which would appear for them not swappable. I don't know how important would this detail be in QA.

Reply:
regarding the dev boards: when the OTP is empty, the idea is to not allow hot swapping as we can't say anything about the production date. we could program some of the QA boards to allow this if needed.